server/apps/main/tests/test_story_filters.py

```diff
@@ -52,25 +50,30 @@ def setUp(self):
         ]
 
     def test_filter_by_tag(self):
-        rendered_string = Template(
-            "{% load story_filters %}"
-            "{{ files|filter_by_tag:'tag2'|length }}"
-        ).render(Context({'files': self.files}))
-
-        self.assertEqual(int(rendered_string), 2)
+        filtered_files = filter_by_tag(self.files, 'tag2')
+        self.assertEqual(len(filtered_files), 2)
 
     def test_filter_by_moderation_status(self):
-        rendered_string = Template(
-            "{% load story_filters %}"
-            "{{ files|filter_by_moderation_status:'approved'|length }}"
-        ).render(Context({'files': self.files}))
-
-        self.assertEqual(int(rendered_string), 2)
+        filtered_files = filter_by_moderation_status(self.files, 'approved')
+        self.assertEqual(len(filtered_files), 2)
 
     def test_filter_in_review(self):
-        rendered_string = Template(
-            "{% load story_filters %}"
-            "{{ files|filter_in_review|length }}"
-        ).render(Context({'files': self.files}))
+        filtered_files = filter_in_review(self.files)
+        self.assertEqual(len(filtered_files), 2)
+        
+
+class TestPaginateMyStories(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
 
-        self.assertEqual(int(rendered_string), 2)
+    def test_paginate_my_stories(self):
```

Comment:
I don't think this currently doesn't actually test whether our pagination works, right? 

As the `Paginator` is being passed a `range(30)` the paginated items are just the numbers 0-29 I think, so the `request` from the factory is in a way superfluous.

Reply:
Ah yes, of course, thanks! I'll work on the test now as part of this PR!

Reply:
Great, thanks!

Reply:
Ok! Added the (hopefully) proper pagination tests.

Reply:
Cool, will have a look now!